cypress/e2e/background-selection.cy.ts

```diff
@@ -22,7 +22,7 @@ describe('Background selector', () => {
       const layers = (<AUTWindowOlMap>window).olMap
         .getLayers()
         .getArray()
-        .filter((l: any) => l.get('cyLayerType') !== 'featureLayer')
+        .filter((l: any) => !/feature(Edit)?Layer/.exec(l.get('cyLayerType')))
```

Comment:
A little pity, we can't target the bgLayers directly during filtering, but I guess that's how it is.